zamba/models/config.py

```diff
@@ -495,19 +496,28 @@ def preprocess_labels(cls, values):
                     labels["site"], proportions=values["split_proportions"]
                 )
             else:
-                logger.info(
-                    "No 'site' column found so videos will be randomly allocated to splits."
-                )
                 # otherwise randomly allocate
-                random.seed(SPLIT_SEED)
-                labels["split"] = random.choices(
-                    list(values["split_proportions"].keys()),
-                    weights=list(values["split_proportions"].values()),
-                    k=len(labels),
+                logger.info(
+                    "No 'site' column found so videos will be randomly allocated using split proportions."
                 )
 
+                expected_labels = [k for k, v in values["split_proportions"].items() if v > 0]
```

Comment:
This var name is a little confusing since `labels` means something specific in our context. Can we change?

Reply:
totes, will use `expected_splits`